extensions/amp-iframe/1.0/component.js

```diff
@@ -61,12 +61,15 @@ export function Iframe({
       // 1. post message is received in viewport
       // 2. exiting viewport
       // This could be optimized by reducing to one call.
-      requestResize(height, width);
+      requestResize(height, width).finally(() => {
+        dataRef.current = null;
```

Comment:
This can be done at the beginning of the callback. No need to wait till the end to clear.

Reply:
Done.